dev/domain-controller/README.md

```diff
@@ -21,6 +21,16 @@ You must have an Azure account and subscription that does not require multi-fact
 
 **NOTE:** To learn how to deploy CAS on Microsoft Azure go to [Deploy Teradici Cloud Access Software on Azure.](https://github.com/teradici/pcoip-agent-azure-templates/blob/master/README.md)
 
+The default method is for a service principal account to be created during deployment. In order to achieve this the Azure administrator must have owner access to the subscription. The administrator must be able to locally login from one of the deployed machines. In some instances this flow will not work, for example accounts with access restrictions or other policies, this requires the service principal account to be manually created. There are multiple ways to manually create a service principal account. See [Creating a Service Principal Account](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal) for instructions on how to do it from the Azure portal. Once the service principal account has been created complete the following steps to integrate it with CAM:
```

Comment:
Thanks - better than what I had! Hmm- this reads like the administrator is logging in themselves when what we're looking for is an account that the CAM deployment scripts can programmatically use to sign in. (Or some such wording.)